JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -73,7 +74,7 @@ ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
 
 		# check for errors
 		anyErrors <- .hasErrors(dataset = dataset, perform = perform,
-								type = c("infinity", "variance"))
+								type = c("infinity", "variance", "observations"), observations.amount = " <= 2")
 
 		doUpdate <- base::identical(anyErrors, FALSE)
```

Comment:
.hasErrors has the argument exitAnalysisIfErrors; when this is set to TRUE it exits the analysis immediately. Consequently you would not need this sort of if-statement. 
See also: [this doc](https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/r-analyses-guide.md#calling-the-error-check-function)

(I don't think you need the doUpdate <- TRUE in the else-statement below this either, as I cannot find another mention of doUpdate elsewhere in your code)